### DISCUSSION TITLE
A doris dialect for sqlalchemy

### DISCUSSION THREAD

**Discussion by actcwlf** [apache/doris#27518] [RFC]
I have been a user of Doris for a while and found it's very useful for me. In my work I need to use doris with python. To support some personal usage, I have made a fork from [pydoris](https://pypi.org/project/pydoris/) and added some features to it ([sqlalchemy-doris](https://github.com/actcwlf/sqlalchemy-doris)). I am not sure whether it is an official extension.  I want to know is there any guideline for community to make contributions or develop tools like these (other than doris itself)

Appreciate your efforts, thanks in advance!

**Comment by bingquanzhao** [comment]
please add my wechat bingquanZHAO. let's talk about it
